src/Turbo/Tests/app/templates/authors.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>Authors</h1>
+
+    <div id="authors" {{ turbo_stream_listen('App\\Entity\\Author') }}></div>
+
+    <turbo-frame id="api">
```

Comment:
Is it important that there is a `turbo-frame` here? I can see you copied it from something existing, but I'd like to only have it if it's necessary.

Reply:
Hm, not AFAIK. It's copied from [books](https://github.com/symfony/ux/blob/2.x/src/Turbo/Tests/app/templates/books.html.twig#L8), so it can probably be removed there as well.